### ISSUE TITLE
iOS home indicator does not automatically hide

### ISSUE THREAD

**Issue by ghost** [jellyfin/jellyfin-ios#188]
When using Jellyfin's app on the full screen of the iPhone, the control bar at the bottom of the iPhone will not automatically hide and disappear when playing a movie

Synology NAS DS720+ Docker Jellyfin 10.6.4
iOS 14.2 iPhone MX MAX and iOS 14.2 iPad pro 10' (2020)
iOS app info: Jellyfin Mobile (iOS) 1.2.0 (5) Expo 38.0.0

![2020-12-13_112605](https://user-images.githubusercontent.com/74287568/102002341-81320780-3d36-11eb-8e7f-94c193d81e2c.jpg)

**Comment by anthonylavado** [comment]
This is known as the "Home Indicator", and is related to something I mentioned in #14.

This will probably not be an issue if we get native playback to be default.

**Comment by superjeng1** [comment]
The problem with native playback is that it doesn't support complex subtitles as I have said in the last comment. Also there is a bug where using native player and embedded subtitles, it would cause double subtitles (A line being displayed twice at the same time). And even if you use separate subtitle files, it will cause it to very briefly display the last line whenever displaying a new line. IMO, even these bugs are fixed, to my knowledge the native player still doesn't support complex subtitles so one may still need the original player.

**Comment by nyanmisaka** [comment]
The native player in iOS only supports webvtt/srt subtitles. Our ass/ssa subtitle renderer will be ignored.

**Comment by superjeng1** [comment]
So, the point I want to make is, getting around this bug by getting native playback to be default is not a great option, and this should be fixed. Sadly I have zero experience on expo so I couldn’t help with this.

**Comment by anthonylavado** [comment]
Right, my original comment is probably best rewritten as "native playback is a possible solution for this **specific** issue". It's not the only solution obviously.

**Comment by thornbill** [comment]
We can probably use this library to achieve this once we eject from Expo, but I think it is blocked until then.

https://github.com/flowkey/react-native-home-indicator

**Comment by superjeng1** [comment]
Hi, I know asking this is kinda counterproductive, but I am still decided to ask. I see couple of issues blocked by eject too so I went to check #134. It seems to me that the only thing left should be the looking into the pipeline, since the upgrade to expo 39 happened already. But then I see the PR is moved to future release. Is there any plan on when will the eject happen?

**Comment by thornbill** [comment]
It doesn't hurt to ask. :slightly_smiling_face:

I just added the comment there about Expo 39 like 2 weeks ago. I saw notes about that in their release notes when I was working on updating it.

You're right the biggest thing we need to work through is how we're going to handle builds and I need to figure out what I'm going to do for a development environment since my primary OS is Linux and you can only develop for iOS in macOS. Expo has let me avoid that issue up till now.

We really don't have timelines since everyone is working on this in their spare time. I'm currently tied up with fixing bugs in web for the 10.7 release, then I will be working on the 0.12 release of Android TV, and then I plan to revisit this. My rough guess would be in 1-2 months I will start working on this, but I have no idea from that point when we would have a release to include it or implement any of the features that are currently blocked.
